shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/analytics/Analytics.kt

```diff
@@ -162,13 +162,6 @@ abstract class Analytics {
         )
     }
 
-    fun toggledPinnedRouteAtStop(pinned: Boolean, routeId: String) {
```

Comment:
This turned out to be redundant with the `togglePinnedRoute` that precedes it; we needed different names so the extensions didn't collide, but since we aren't defining each page's analytics as a separate extension on the `AnalyticsProvider` anymore, we do not need two equivalent versions of methods like this. I didn't need to make this part of this PR, but it felt like it wasn't worth spinning out into a separate PR.